src/index.js

```diff
@@ -139,6 +139,15 @@ const applyConfig = (config) => {
   // Custom block-style colors
   config.settings.available_colors = eea.colors;
 
+  // Site theme colors
+
+  config.settings.themeColors = [
+    { value: 'default', title: 'Default' },
```

Comment:
I think this can be removed as the select widget already adds a `No value` default option

Reply:
Yes, but I'm using the theme colorpicker widget, not the select anymore https://github.com/eea/volto-widget-theme-picker/blob/develop/src/Widgets/ThemePicker.jsx that's why I'm inserting default also